Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -410,6 +413,8 @@ BitSet32 VertexManagerBase::UsedTextures() const
 
 void VertexManagerBase::Flush()
 {
+  LWDProfiler::ScopedMarker marker("Vertex/Flush");
```

Comment:
I think it's best to keep only this marker and remove all other Vertex/*

Calc ZSlope and Calc Binormal will always be cheap. And Draw Batch is a bit of a wasted profile because it's almost always called exactly once per flush.